### ISSUE TITLE
Cannot run static version of Butler

### ISSUE THREAD

**Issue by labbydev** [palantirnet/butler#7]
I tried just running Butler statically to see if it would pull up the **sculpin** prototype. It seems to have installed the dependencies, however when I ran `gulp` this is the error that I got:

```
Error: Cannot find module 'isarray'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/LaurenByrwa/Sites/butler/node_modules/gulp-inject/node_modules/gulp-util/node_modules/multipipe/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js:25:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/LaurenByrwa/Sites/butler/node_modules/gulp-inject/node_modules/gulp-util/node_modules/multipipe/node_modules/duplexer2/node_modules/readable-stream/readable.js:1:90)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

**Comment by labyrwa** [comment]
When trying to just run a static version of the **jekyll** branch I also got an error.

```
Error: Cannot find module 'process-nextick-args'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/LaurenByrwa/Sites/butler/node_modules/gulp-compass/node_modules/gulp-util/node_modules/through2/node_modules/readable-stream/lib/_stream_duplex.js:20:23)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Comment by patrickfweston** [comment]
In an interesting turn of events, I'm able to get the **sculpin** version of butler up and running on a static site. I did have to make a few modifications, but I never really got any error messages that were too scary (or anything like I got the last time I tried to run it with Drupal). 

I simply cloned the repo and checked out the `sculpin-starter` branch. I then tried to run the sculpin command to start the server `sculpin generate --watch --server`. This worked except for the fact that it was looking for a _posts_ content type, which doesn't exist. 

I then ran `gulp` which worked and started serving up the page. One thing that did not work was that any page other than the index.html page did not include _styles.css_. It seems that the CSS injection is giving a relative path to the CSS file, and since any of the links point inside of the /styleguide folder, the path of the CSS file cannot be resolved.

To solve this, I changed the Gulpfile.js to use an absolute path by changing _addRootSlash_ to **true**:
```javascript
return target.pipe(inject(sources, {ignorePath: 'source', addRootSlash: *true* }))
    .pipe(gulp.dest('source/code/_views'));
```
(found on line 32 in Gulpfile.js...I'll try to add a line comment for the code)

Once this change has been made, rerunning `gulp` produces the styleguide and everything looks alright to me. 

Just for clarity's sake, I'm _not_ running this inside of a Vagrant box.
